npa.js

```diff
@@ -48,8 +48,10 @@ function npa (arg, where) {
     const valid = validatePackageName(arg)
     if (valid.validForOldPackages) {
       name = arg
-    } else {
+    } else if (valid.validForNewPackages) {
```

Comment:
```suggestion
    else if (arg)
      spec = arg
    else
      throw invalidPackageName(arg, valid)
```

Reply:
Another possibility, since `""` is a valid path resolving to the current working directory, is that we just turn this into `file:`.

```suggestion
  } else {
    const valid = validatePackageName(arg)
    if (valid.validForOldPackages)
      name = arg
    else if (arg)
      spec = arg
    else
      spec = 'file:'
  }
```